changelog.txt

```diff
@@ -1,3 +1,6 @@
+0.79
+ * RealmResults.remove(index) and RealmResults.removeLast() are now working again.
```

Comment:
I would rephrase it:
 * Re-enabled RealmResults.remove(index) and RealmResults.removeLast()